### PR TITLE
[SPIKE] Improve backend documentation

### PR DESCRIPTION
**Pull request by travis-newby**
This PR improves a fair bit of project documentation. It updates things like CODEOWNERS and MAINTAINERS to reflect the current project makeup, and fixes a few minor issues with the project readme and contribution docs. It also standardizes around a "note" format, and hopefully makes things a lot easier to read and follow. 

The biggest changes center around the data pipeline docs. This PR updates the data pipeline readme, and separates out the data pipeline installation steps into their own file. It also updates the DATASETS file to be way more comprehensive (thanks Sam!).

### CODE REVIEW

File: INSTALLATION.md [hunk 1]
```diff
@@ -2,85 +2,111 @@
...
-## Install Docker
+---
 
-Follow the [Docker installation
+### 5. Install Docker (Optional)
+
+Using Docker is optional; the Frontend Client and the Data Pipeline and Scoring Application can be run without Docker. If you wish to install Docker, follow the [Docker installation
 instructions](https://docs.docker.com/get-docker/) for your platform.
+
+## Next Steps
```
Comment (vim-usds) [inline INSTALLATION.md:104]: Nice touch with the table

File: data/data-pipeline/INSTALLATION.md [hunk 2]
```diff
@@ -0,0 +1,135 @@
+# Justice40 Data Pipeline and Scoring Application Installation Guide
+
+This page documents the local environment setup steps for the Justice40 Data Pipeline and Scoring Application. It covers steps for macOS and Win10. If you are not on either of those platforms, install the software using instructions appropriate for your operating system and device.
+
+> :warning: **WARNING**  
+> This guide assumes you've performed all prerequisite steps listed in the [main installation guide](/INSTALLATION.md). If you've not performed those steps, now is a good time.
+
+> :bulb: **NOTE**
```
Comment (vim-usds) [inline data/data-pipeline/INSTALLATION.md:8]: I'm digging these bulbs and warning :)

**Review by vim-usds — APPROVED** [review]
Thank you Travis, way to make this super cohesive!

File: data/data-pipeline/INSTALLATION.md (hunk 2 above)
Comment (travis-newby) [reply data/data-pipeline/INSTALLATION.md:8]: Thanks!

**Pull request merged**